docs/concepts/configuration/manage-compute-resources-container.md

```diff
@@ -231,8 +231,8 @@ Allocated resources:
   680m (34%)      400m (20%)    920Mi (12%)        1070Mi (14%)
 ```
 
-In the preceding output, you can see that if a Pod requests more than 1120m
-CPUs or 6.23Gi of memory, it will not fit on the node.
+In the preceding output, you can see that if a Pod requests more than 1800m
```

Comment:
The change is incorrect. Please note that 680m CPU and 920Mi memory have already been allocated on the node.

Reply:
ok,thanks,I will close the pr!